src/plugins/intel_gpu/src/kernel_selector/core/cl_kernels/include/batch_headers/fetch_weights.cl

```diff
@@ -419,6 +419,78 @@ inline uint get_giy_xs_os_xsv2_osv_index(uint g, uint o, uint i, uint y, uint x,
         CAT(prefix, _OFFSET),                                                   \
         sub_group_size)
 
+inline uint get_is_os_yx_isa8_osv8_isv2_index(uint o, uint i, uint y, uint x, uint size_x,
+                                                uint size_y, uint size_ifm, uint size_ofm, uint offset)
+{
+	const uint isv2_idx = i % 2;
```

Comment:
Is it really necessary to use such specific code alignment style?

Reply:
updated it.